app/utils/uuid_utils.rb

```diff
@@ -0,0 +1,12 @@
+module UUIDUtils
+  module_function
+
+  def bin16_uuid
+    # TODO: Use https://github.com/assaf/uuid
+    s = SecureRandom.uuid.gsub("-", "")
```

Comment:
Did you @kpuputti check somewhere that these are v1 UUIDs? My understanding has been they'd be random (v4), which doesn't work for clustering by time.